spiel/st/spiel_multiple_confd.sh

```diff
@@ -154,11 +154,11 @@ confd_mkfs_start() {
     -m $MAX_RPC_MSG_SIZE -q $TM_MIN_RECV_QUEUE_LEN -c $CONF_FILE\
     -w 3 -f ${!fid}"
 
-    echo "--- `date` ---" >>$path/m0d.log
-    cd $path
+    echo "--- `date` ---" >>"$path"/m0d.log
+    cd "$path"
 
-    echo $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS
-    $M0_SRC_DIR/utils/mkfs/m0mkfs $OPTS >>$path/mkfs.log ||
+    echo "$M0_SRC_DIR"/utils/mkfs/m0mkfs "$OPTS"
```

Comment:
$OPTSis a list of arguments to be passed to the command 'm0mkfs'
If it is quoted, the list of arguments will be passed to 'm0spiel' as a single argument.
Please refer to:
https://github.com/Seagate/cortx-motr/pull/1905
https://github.com/Seagate/cortx-motr/pull/1730

Reply:
Should I remove this reference ?

Reply:
yes.

Reply:
Done